k8s/manifests/charts/ck-loadbalancer/templates/cilium/lb-ip-pool.yaml

```diff
@@ -1,4 +1,6 @@
-{{- if .Values.ipPool.cidrs }}
+{{- if .Values.l2.enabled }}
```

Comment:
Why'd we go from `ipPool.cidrs` to `l2.enabled` ?

Reply:
Fixed